### PR TITLE
Granary on b0.20.1

### PR DESCRIPTION
**Pull request by maaktweluit**
Duplicate PR from #4276 so the test_e2e_source_smoke can run on this release branch

### CODE REVIEW

File: scripts/node_integration_tests/tests/base.py [hunk 1]
```diff
@@ -20,40 +21,36 @@
...
 
 
-KEYSTORE_DIR = 'rinkeby/keys'
-
-
 def disable_key_reuse(test_function: Callable) -> Callable:
     @wraps(test_function)
     def wrap(*args, **kwargs) -> None:
-        args[0].reuse_keys = False
+        reuse_keys = args[0].reuse_keys
```
Comment (mfranciszkiewicz) [inline scripts/node_integration_tests/tests/base.py:27]: We might be more explicit with what `args[0]` is by naming (and type annotating) the first argument

File: scripts/node_integration_tests/tests/base.py [hunk 2]
```diff
@@ -68,7 +65,7 @@ def _run_test(self, test_path: str, *args, **kwargs):
             self.datadirs[node_id] = datadir
             os.makedirs(datadir)
 
-        cwd = pathlib.Path(__file__).resolve().parent.parent
+        cwd = Path(__file__).resolve().parent.parent
```
Comment (mfranciszkiewicz) [inline scripts/node_integration_tests/tests/base.py:68]: When `base.py` moves to a lower or upper directory level this line will also have to change.

I don't know the general assumptions of this test suite, but if it's not costly (and possible) to 
`from node_integration_tests import run_test`
then this line could be changed to
`cwd = Path(run_test.__file__).resolve().parent`
which would be more human-error proof and self-explanatory.

**Review by mfranciszkiewicz — APPROVED** [review]
Some extra suggestions

**Comment by maaktweluit** [comment]
@mfranciszkiewicz thanks for the review! 
will add those suggestions in a separate PR to develop, would like to keep this PR the same as the code that is tested for 2 weeks on buildbot

**Comment by tworec** [comment]
it still doesn't work for my PR #4177 
```
error: unrecognized arguments: --granary-hostname golem-granary
```
https://buildbot.golem.network/buildbot/#/builders/16/builds/72

**Comment by maaktweluit** [comment]
@tworec Did you fetch before merging? it looks like this PR is not included in your branch

```
$ git log --graph --oneline
...
*   64869c26b Merge branch 'b0.20' into multi-WTCT-support
|\
| * fe81243b4 Fix HyperV warning events (#4353)
```

```
$ git log fe81243b4
...
Date:   Mon Jun 17 20:24:38 2019 +0200

    Fix HyperV warning events (#4353)
```

**Comment by tworec** [comment]
true, I've merged local b0.20.1 into my PR and it was not synced with origin/b0.20.1 which have already your changes... Sorry for bugging you :-)

**Pull request merged**